.nvmrc

```diff
@@ -1 +1 @@
-16.13.1
```

Comment:
Unrelated but we should probably update the template here too, OOS of this PR of course :P

Reply:
The plugin starter template has much more to update, like getting rid of babel, going full typescript, updating nodejs etc.. I don't think we have the resources right now for it but when time comes.